Program/Seasons.py

```diff
@@ -0,0 +1,220 @@
+import time
+
+
+
+#This is the stored pictures, calling "global" so the entire program can use them.
+setMediaPath("F:\COMP 120 Tinkering Graphics-JH182233\Pictures") 
+picfour = makePicture("Tree-Four-Seasons.jpg") 
+picwinter = makePicture("Tree-Winter.jpg") 
+picspring = makePicture("Tree-Spring.jpg") 
+picsummer = makePicture("Tree-Summer.jpg")
+picautumn = makePicture("Tree-Autumn.jpg")  
+picM = makeEmptyPicture(513, 556, black)
+pic = makeEmptyPicture(1025, 1110, black)
+
+
+#The main function for the program
+def main():
+#create an empty picture and populate it with others 
+    startX = 0
+    startY = 0 
+    repaint (pic)
+    time.sleep(1)  
+    copyInto(picspring, pic, startX, startY)
+    repaint (pic)
+    
+    startX = 512
+    startY = 0    
+    time.sleep(1) 
+    copyInto(picsummer, pic, startX, startY)
+    repaint (pic)
+    
+    startX = 0
+    startY = 554  
+    time.sleep(1)  
+    copyInto(picautumn, pic, startX, startY)
+    repaint (pic)
+    
+    startX = 512
+    startY = 554  
+    time.sleep(1)  
+    copyInto(picwinter, pic, startX, startY)
+    repaint (pic)
+    request()
+    
+    
+    
+#Now the program will ask for a season that you would like to view
+def request():
+    startX = 230
+    startY = 300   
+    season = requestString("What season would you like to activate?")
+    if String(season) =="Winter":
+       blackoutleft()
+       copyInto(picwinter, pic, startX, startY)
+       repaint(pic)
+       time.sleep(2)
+       request2()
+    elif String(season) =="Spring":
+       blackoutleft()  
+       copyInto(picspring, pic, startX, startY)
+       repaint(pic)
+       time.sleep(2)
+       request2()
+    elif String(season) =="Summer":
+       blackoutleft()   
+       copyInto(picsummer, pic, startX, startY)
+       repaint(pic)
+       time.sleep(2)
+       request2()
+    elif String(season) =="Autumn":
+       blackoutleft()   
+       copyInto(picautumm, pic, startX, startY)
+       repaint(pic)
+       time.sleep(2)
+       request2()
+    else:
+       blackouttop()
+       request2()
+       time.sleep(2)
+     
+    
+
+#another request, a simple Y/N question to see if you want to continue, it will either end for "N" or continue if "Y"    
+def request2():
+    endSeason = requestString("Would you like to view more Y/N?")
+    if String(endSeason) == "Y":
+       copyInto(picfour, pic, 0, 0)
+       repaint(pic)
+       playWinter()
+    elif String(endSeason) == "N":
+       blackouttop()
+       copyInto(picfour, pic, 0, 0)
+       repaint(pic)
+       showInformation("Thank you for using BioSeasons")
+       blackouttop()
+    else:       
+       request2()
+     
+     
+
+#Thie function will play through the seasons while calling on the merge function to produce a better flow.
+def playWinter():
```

Comment:
This function could have more appropriate naming. It is called playWinter, but it appears to play all of the seasons, starting at winter